src/plugins/recordTypes/collectionobject/forms/public.jsx

```diff
@@ -0,0 +1,169 @@
+            <Field name="objectNameType" />
+            <Field name="objectNameLanguage" />
+            <Field name="objectNameNote" />
+          </Field>
+        </Field>
+
+      </Panel>
+
+      <Panel name="desc" collapsible>
+        <Field name="colors">
+          <Field name="color" />
+        </Field>
+
+        <Field name="materialTechniqueDescription" subpath="ns2:collectionobjects_fineart" />
+
+        <Field name="materialGroupList">
+          <Field name="materialGroup">
+            <Field name="material" />
+            <Field name="materialComponent" />
+            <Field name="materialComponentNote" />
+            <Field name="materialName" />
+            <Field name="materialSource" />
+          </Field>
+        </Field>
+
+        {extensions.dimension.form}
+
+        <Panel name="content" collapsible>
+          <Field name="contentConcepts">
+            <Field name="contentConcept" />
+          </Field>
+        </Panel>
+      </Panel>
+
+      <Panel name="prod" collapsible>
+        <Row>
+          <Col>
+            <Field name="objectProductionDateGroupList">
+              <Field name="objectProductionDateGroup" />
+            </Field>
+
+            <Field name="objectProductionPlaceGroupList">
+              <Field name="objectProductionPlaceGroup">
+                <Field name="objectProductionPlace" />
+                <Field name="objectProductionPlaceRole" />
+              </Field>
+            </Field>
+          </Col>
+
+          <Col>
+            <Field name="objectProductionPeopleGroupList">
+              <Field name="objectProductionPeopleGroup">
+                <Field name="objectProductionPeople" />
+                <Field name="objectProductionPeopleRole" />
+              </Field>
+            </Field>
+
+            <Field name="objectProductionPersonGroupList">
+              <Field name="objectProductionPersonGroup">
+                <Field name="objectProductionPerson" />
+                <Field name="objectProductionPersonRole" />
+              </Field>
+            </Field>
+
+            <Field name="objectProductionOrganizationGroupList">
+              <Field name="objectProductionOrganizationGroup">
+                <Field name="objectProductionOrganization" />
+                <Field name="objectProductionOrganizationRole" />
+              </Field>
+            </Field>
+          </Col>
+        </Row>
+      </Panel>
+
+      <Panel name="hist" collapsible>
+        <Field name="objectHistoryNote" />
+      </Panel>
+
+      <Panel name="owner" collapsible>
+        <Field name="ownersContributionNote" />
+      </Panel>
+
+      <Panel name="viewer" collapsible>
+        <Field name="viewersContributionNote" />
+      </Panel>
+    </Field>
+  );
+};
+
+export default (configContext) => ({
+  messages: defineMessages({
+    name: {
+      id: 'form.collectionobject.public.name',
+      defaultMessage: 'Public Browser Template',
+    },
+  }),
+  sortOrder: 3,
```

Comment:
Since this is overriding the `public` form from cspace-ui.js, but the `messages` and `sortOrder` are not being changed, they can be omitted here.